game/src/plugins/entity/spawn/src/spawn.kt

```diff
@@ -13,3 +13,7 @@ object Spawns {
 fun npc_spawn(name: String, x: Int, y: Int, id: Int? = null) {
```

Comment:
See https://kotlinlang.org/docs/reference/functions.html#default-arguments for information on default args